linode/manager
# Refactor: [M3-6522-selection-card] Selection Card: named exports and v7 story

## Description 📝
This PR refactors the `SelectionCard` component exports and handles the new way of doing stories. It does not modify the component in any way, or at least shouldn't. Part of our epic to migrate away from .mdx files.

## Preview 📷
No visual diff

## How to test 🧪
1. Confirm no regression on the SelectionCard component
2. Confirm and review new story at /?path=/docs/components-selectioncard--documentation
